types/suman/suman-tests.ts

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
It will need tests to ensure that code like https://github.com/sumanjs/suman#simple-example works.

Reply:
what's wrong with console.log()? :) this is just a dummy PR, so that I know what to expect. An IMPORTANT QUESTION: Can I really not name any interface starting with an I? As in `interface IFooBar {}`... that is a really restrictive and unexpected limitation that forces me to refactor a lot of code :(